src/Client/OfficeInterop/OfficeInterop.fs

```diff
@@ -1791,6 +1833,43 @@ let addBuildingBlockAt (excelIndex: int) (newBB: CompositeColumn) (table: Table)
         |> List.iteri (fun ci header -> ExcelHelper.addColumnAndRows (float (excelIndex + ci)) table header bodyValues.[ci] |> ignore)
     }
 
+type BodyCellType =
+| Term
+| Unitized
+| Text
+| Data
+
+let getSelectedCellType () =
+    Excel.run(fun context ->
+        promise {
+            let! result = tryGetActiveAnnotationTable context
+
+            match result with
+            | Result.Ok excelTable ->
+                let! _, rowIndex = getSelectedBuildingBlockCell excelTable context
+                let! arcTable = ArcTable.tryGetFromExcelTable(excelTable, context)
+                let! (arcMainColumn, _) = getArcMainColumn excelTable arcTable.Value context
+
+                if (arcMainColumn.Validate false) && rowIndex > 0 then
+
+                    let result =
+                        match arcMainColumn with
+                        | amc when amc.Header.ToString() = "Output [Sample Name]" -> None
```

Comment:
I might not understand this part, but it seems to be very bad design to match to specific strings if we want to rely on ARCtrl for newest ARC specification changes! 

In addition, for example how is `Output [Material]` handled?

Reply:
fixed